### ISSUE TITLE
add social_facility=*

### ISSUE THREAD

**Issue by matkoniecz** [streetcomplete/StreetComplete#3939]
### General
Affected tag(s) to be modified/added: [social_facility](https://wiki.openstreetmap.org/wiki/Key:social_facility)
Question asked: **Which kind of social facility it is?**

### Checklist
Checklist for quest suggestions (see [guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md)):
- [x] 🚧 To be added tag is established and has a useful purpose
- [x] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one)
- [x] 🐿️ Easily answerable by everyone from the outside but a survey is necessary
- [x] 💤 Not an overwhelming percentage of quests have the same answer (No spam)
- [x] 🕓 Applies to a reasonable number of map data (Worth the effort) https://taginfo.openstreetmap.org/tags/amenity=social_facility#combinations 

### Ideas for implementation

**Element selection:** https://overpass-turbo.eu/s/1hkd 

**Metadata needed:** should not be needed,  Custom per-country order is likely an overkill: remembering latest used should be good enough

**Proposed UI:**

[Value](https://wiki.openstreetmap.org/wiki/Key:social_facility) selection + option to mark that it is office managing social facility, not a social facility itself.

**Comment by westnordost** [comment]
I think this quest would result in many notes being created because the range of possible things tagged as social facilities is much greater than currently documented and sometimes quite hard to ascertain or define exactly (a lot of overlap and/or "not quite" X)

**Comment by matkoniecz** [comment]
> much greater than currently documented

That, I think, is not a problem by itself - in such case it would reveal missing tagging which could be added

**Comment by westnordost** [comment]
Additionally, I think there would be many different available selections because there is a big range of social facilities. Displaying this as a list of radiobuttons will likely not be feasible (too long), also it feels kind of like duplicate translator work to localize all the different social facilities in StreetComplete but then (later) also in the iD presets.

Maybe there should be a new generic interface that is similar to the "What is here now?" dialog which just searches through iD presets (but limits results to social facilities, in this case).

**Comment by matkoniecz** [comment]
> Maybe there should be a new generic interface that is similar to the "What is here now?" dialog which just searches through iD presets (but limits results to social facilities, in this case).

I agree, this would avoid duplicate translation work and finding missing POIs would result in improving also iD (though it has a small negative that adding missing features would need to happen upstream).

**Comment by bgo-eiu** [comment]
`social_facility:for` is unfortunately poorly defined at the moment. A social facility doesn't have to be for a group of people or target demographic; it can have a descriptive cause or purpose that could apply to anyone. Some of the existing uses of the tag are kind of shoehorning people-words into the tag where it does not seem like a great fit. For example, `social_facility:for=victim` does not really describe how the purpose of the social facility well - usually social facilities like this will be for addressing domestic violence broadly, or for helping people with PTSD, where the people being "victims" of something is not really the important part.

Even though it is common colloquially for people to use the term "homeless person," that is also not really a "type" of person. Homelessness is a state a person can be in, they are still the same person before and after they experience homelessness, so something like `social_facility:for=housing` would make more sense since the function of the facility would be to provide housing assistance to anybody who needs it, rather than provide general assistance to a type of person who does not have a house.

I am not sure of the solution to this, just pointing out it is a messy tag. I ended up using `social_facility:for=violence_interruption` recently for a facility that does outreach services to potential offenders as a preventative measure against violent crime. "Violence interruption" is a formal term for this service model, so it seems appropriate, whereas coming up with something lime `social_facility:for=potential_offender` just seems wrong (they're not trying to accuse people of "pre-crimes," and the work is really for the benefit of the community at large, not a type of person).

**Comment by matkoniecz** [comment]
Note social_facility:for=* is/was discussed in #3940 which was rejected

this is for something else

**Comment by bgo-eiu** [comment]
OK, I think I misunderstood. When I clicked on that I assumed it was closed to discuss here rather than rejected

**Comment by matkoniecz** [comment]
To be even more exact, as mentioned in https://github.com/streetcomplete/StreetComplete/issues/3940#issuecomment-1101313338 this will also may end tagging `social_facility:for` - but only together with `social_facility` and only where they end describing well defined objects.
